README.md

```diff
@@ -3,38 +3,31 @@
 ## Zu installierende Node.js-Pakete
 npm install dotenv
 npm install otpauth
```

Comment:
```suggestion
npm install dotenv otpauth
```
Die Packages können in einem Befehl installiert werden. Müssen die global installiert werden für irgendwelche Aufrufe oder gehören die in die package.json?

Reply:
otpauth brauchen wir erst später für die 2FA. otpauth  nehme ich jetzt erstmal wieder raus. dotenv sollte lokal installiert werden (recommended). Die dotenv  wird dann automatisch in die package.json eingetragen.

Reply:
Dann schlage ich vor, dass es aus der Readme raus kommt und wir es lokal installieren, damit es in der package.json drin ist. Dann wird es ja ohnehin immer mitinstalliert